ui/scripts/check-raw-palette.mjs

```
@@ -178,74 +251,122 @@ function clientApps() {
 
-if (errors.length > 0) {
-  console.error('check-raw-palette: FAILED\n');
-  for (const error of errors) console.error(`  - ${error}`);
-  process.exit(1);
+function main() {
+  const { errors, scanned, apps } = runCheck();
+
+  if (errors.length > 0) {
+    console.error('check-raw-palette: FAILED\n');
+    for (const error of errors) console.error(`  - ${error}`);
+    process.exit(1);
```

ESLint

[error] 'process' is not defined.

(no-undef)